bentoml/frameworks/spark_mllib.py

```diff
@@ -0,0 +1,85 @@
+import importlib
+import json
+import logging
+import os
+
+from bentoml.exceptions import BentoMLException, MissingDependencyException
+from bentoml.service import BentoServiceArtifact, BentoServiceEnv
+
+
+class PySparkModelArtifact(BentoServiceArtifact):
+    def __init__(self, name: str):
+        super().__init__(name)
+        self._model = None
+        self._sc = None
+
+    def set_dependencies(self, env: BentoServiceEnv):
+        env.add_pip_package("pyspark")
```

Comment:
Don't we need something like `env.add_conda_dependency("openjdk")` to make this work?